### PR TITLE
feat(Services endorsements api): Endorsements email pdf

### PR DESCRIPTION
**Pull request by rafnarnason**
# ...

Attach a link to issue if relevant

## What

adding support to email pdf endorsements

## Why

part of the service offering - and emails rock

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { IsBoolean, IsEmail, IsNotEmpty } from 'class-validator'
```
Comment (kksteini) [inline apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts:2]: IsEmail unused import?

File: apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts [hunk 2]
```diff
@@ -380,4 +388,119 @@ export class EndorsementService {
...
+            },
+            {
+              component: 'Button',
+              context: {
+                copy: 'Takki',
+                href: 'http://www.island.is',
+              },
+            },
+            { component: 'Copy', context: { copy: 'Með kveðju,' } },
+            { component: 'Copy', context: { copy: 'TEST' } },
```
Comment (kksteini) [inline apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts:490]: This looks like test code.
Will this become visible to regular users if it were to go on prod?

**Review by kksteini — APPROVED** [review]
LGTM but I have a question

File: apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts (hunk 1 above)
Comment (rafnarnason) [reply apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts:2]: well spotted - will cleanup

File: apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts (hunk 2 above)
Comment (rafnarnason) [reply apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts:490]: the whole app is not visible on production - so this is me getting it to dev staging and finally to production to see if the emailing functionality works on those stages

File: apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts (hunk 1 above)
Comment (kksteini) [reply apps/services/endorsements/api/src/app/modules/endorsement/dto/sendPdfEmail.response.ts]: :+1:

File: apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts (hunk 2 above)
Comment (kksteini) [reply apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts:490]: :+1:

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**